### DISCUSSION TITLE
Discord voice chat "Connecting to RTC"

### DISCUSSION THREAD

**Discussion by ktr-man** [hufrea/byedpi#185] [General]
Возможно ли сделать поддержку RTC протокола в Discord? На данный момент единственный вариант - zapret-discord только под windows и не работает как прокси.

**Comment by Fidanze** [comment]
судя по тому что после подключения к голосовому чату в дисе через впн, можно отключиться и спокойно общаться дальше, скорее всего блокируют только "рукопожатие" во время которого компы обмениваются  своими адресами. оно проходит через сервер дискорда, поэтому возможно достаточно будет обойти блокировку рукопожатия (точно не знаю)

**Comment by Fidanze** [comment]
также вроде работает метод обхода через warp тунель в  amneziawg

**Comment by ktr-man** [comment]
> также вроде работает метод обхода через warp тунель в amneziawg

Работает, но это буквально использование чужого прокси сервера. Сейчас через него куча народу сидит и производительность/надёжность оставляет желать лучшего. Не говоря уже о том, что если перекроют - мало что поделаешь.
